jas/common.go

```diff
@@ -270,6 +270,25 @@ func convertToFilesExcludePatterns(excludePatterns []string) []string {
 	return patterns
 }
 
+func CheckForSecretValidation(xrayManager *xray.XrayServicesManager, validateSecrets bool) bool {
+	// Ordered By importance
+	// first check for flag
+	if validateSecrets {
+		return validateSecrets
+	}
+	// second check for env var
+	if strings.ToLower(os.Getenv("JF_VALIDATE_SECRETS")) == "true" {
+		return true
+	}
+	// third check for platform api
+	isEnabled, err := xrayManager.IsTokenValidationEnabled()
+	if isEnabled && err == nil {
+		return true
+	}
+	// at this point secret validation will be false
+	return false
```

Comment:
```suggestion
	isEnabled, err := xrayManager.IsTokenValidationEnabled()
    return err == nil && isEnabled
```
first check err, can be shorten